### PR TITLE
color picker module added

### PR DESCRIPTION
**Pull request by naveen17797**
- color picker experimental module converted to stable
- now user can pick primary and secondary color on any theme
screenshot below
screenshot from 2018-02-28 22-44-02

### CODE REVIEW

**Comment by naveen17797** [comment]
@teryhill @aethelwulffe @pri2si17-1997 please review this.

File: interface/themes/style.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<?php 
+/** 
+ * function to apply selected color in the color picker
+ * 
+ * Copyright (C) 2017 Art Eaton
```
Comment (teryhill) [inline interface/themes/style.php]: Change this to 2018 and Your name

**Comment by naveen17797** [comment]
as the user selects color picker, jquery is used to reflect the color changes live.

File: interface/themes/style.php (hunk 1 above)
Comment (aethelwulffe) [reply interface/themes/style.php]: How the heck did my name get in there???

**Comment by aethelwulffe** [comment]
OK guys, I have tried it, and it is cool and everything, but @naveen17797 this has been a WIP, so I want to hear when you think it is ready for merging.  I feel we can merge it as an ongoing project if it isn't broken in any way.  It doesn't endanger anything.  I think the most important thing to consider for something that may change soon is that you are SURE that the global variables you are using are the ones that you want.  These could be updated or removed by an upgrade too, but they are the only things that really affect the database entries.

**Comment by naveen17797** [comment]
Yes ,i will let you know when it is ready for merge, and will make sure that it didnt break any ui elements before making a merge.

**Comment by naveen17797** [comment]
You can merge if you want, it doesnt affect any backend operations, im afraid it may affect ui elements.

**Comment by aethelwulffe** [comment]
Style selection in the menu might benefit from a "Experimental" tag.  Not a huge worry though.

**Comment by naveen17797** [comment]
As you said merging might help to adopt theme setter faster, i think then it is ready for merge.

**Comment by tmccormi** [comment]
@naveen17797 you have a conflict that needs to be fixed before I can merge this one.

**Comment by naveen17797** [comment]
resolved conflicts

**Pull request merged**